languageserver/integration/commands.go

```diff
@@ -208,14 +210,70 @@ func (i *FlowIntegration) sendTransaction(conn protocol.Conn, args ...interface{
 		return nil, fmt.Errorf("failed to parse JSON arguments")
 	}
 
-	signer, err := i.state.EmulatorServiceAccount()
+	serviceAccount, err := i.state.EmulatorServiceAccount()
 	if err != nil {
 		return nil, fmt.Errorf("failed to get service account, err: %w", err)
 	}
-	signer.SetAddress(flow.HexToAddress(signers[0]))
 
-	const gasLimit uint64 = 1000
-	_, txResult, err := i.sharedServices.Transactions.Send(signer, code, "", gasLimit, txArgs, "")
+	payer := serviceAccount.Address()
```

Comment:
Maybe a better naming would be `serviceAddress`

Reply:
I wanted to signify the fact that service account will pay for it :)